MessageBird/ConversationsClient.cs

```diff
@@ -0,0 +1,129 @@
+using MessageBird.Net;
+using MessageBird.Objects.Conversations;
+using MessageBird.Resources.Conversations;
+using MessageBird.Utilities;
+
+namespace MessageBird
+{
+    public partial class Client
+    {
+        public ConversationList ListConversations(int limit = 20, int offset = 0)
+        {
+            var resource = new ConversationLists();
+
+            var list = (ConversationList) resource.Object;
+            list.Limit = limit;
+            list.Offset = offset;
+
+            restClient.Retrieve(resource);
+
+            return resource.Object as ConversationList;
+        }
+
+        public Conversation ViewConversation(string id)
+        {
+            ParameterValidator.IsNotNullOrWhiteSpace(id, "id");
+            var resource = new Conversations(new Conversation {Id = id});
+            restClient.Retrieve(resource);
+
+            return resource.Object as Conversation;
+        }
+
+        public Conversation StartConversation(ConversationStartRequest startRequest)
+        {
+            ParameterValidator.IsNotNullOrWhiteSpace(startRequest.To, "to");
+            ParameterValidator.IsNotNullOrWhiteSpace(startRequest.ChannelId, "channelId");
+            ParameterValidator.IsNotNull(startRequest.Type, "type");
+            ParameterValidator.IsNotNull(startRequest.Content, "content");
+            var resource = new ConversationStart(startRequest);
+            restClient.Create(resource);
+
+            return resource.Object as Conversation;
+        }
+
+        public Conversation UpdateConversation(string id, Conversation conversation)
+        {
+            ParameterValidator.IsNotNullOrWhiteSpace(id, "id");
+            conversation.Id = id;
+            var resource = new Conversations(conversation);
+            
+            restClient.Update(resource);
+
+            return resource.Object as Conversation;
+        }
+
+        public MessageList ListConversationMessages(string conversationId, int limit = 20, int offset = 0)
```

Comment:
I'm thinking it may be a good idea to rename `Message` and `MessageList` to `ConversationMessage` and `ConversationMessageList`. It is rather verbose but within the scope of this library, "Message" is pretty ambiguous. What do you think?